components/transactions/filters/TransactionsKindFilter.js

```diff
@@ -77,10 +77,11 @@ const REACT_SELECT_COMPONENT_OVERRIDE = {
   MultiValue: () => null, // Items will be displayed as a truncated string in `TruncatedValueContainer `
 };
 
-const TransactionsKindFilter = ({ onChange, value, ...props }) => {
+const TransactionsKindFilter = ({ onChange, value, kinds, ...props }) => {
   const intl = useIntl();
   const getOption = (value, idx) => ({ label: i18nTransactionKind(intl, value), value, idx });
-  const options = React.useMemo(() => Object.values(DISPLAYED_TRANSACTION_KINDS).map(getOption), [intl]);
+  const displayedKinds = kinds || Object.values(DISPLAYED_TRANSACTION_KINDS);
+  const options = displayedKinds.map(getOption);
```

Comment:
When tested with prod snapshot I got a `null` option from the API here, probably from old transactions.
I suggesting filtering out options that are null.
```suggestion
  const options = displayedKinds.filter(kind => kind !== null).map(getOption);
```

Select with empty option bug:
![Screenshot from 2021-09-06 11-11-04](https://user-images.githubusercontent.com/2119706/132230534-8562937e-072b-48d9-bfa6-75724050bef4.png)

GraphQL response:
![Screenshot from 2021-09-06 11-14-41](https://user-images.githubusercontent.com/2119706/132230602-f263c5e9-adcf-495a-a8b1-4b0a8c29f43f.png)

Reply:
Thank you! Will patch the API.

Reply:
Done